tests/dialects/test_bigquery.py

```diff
@@ -187,6 +100,16 @@ def test_bigquery(self):
             )
             assert "'END FOR'" in cm.output[0]
 
+        self.validate_identity("CREATE SCHEMA x DEFAULT COLLATE 'en'")
+        self.validate_identity("CREATE TABLE x (y INT64) DEFAULT COLLATE 'en'")
+        self.validate_identity("PARSE_JSON('{}', wide_number_mode => 'exact')")
+        self.validate_identity("FOO(values)")
+        self.validate_identity("STRUCT(values AS value)")
```

Comment:
The problem raised by the issue would also occur with any function, because function args are parsed using `_parse_lambda` which ends up calling `_parse_select_or_expression` and previously we were hitting the branch where the `VALUES` clause parsing happens. Hence the recently [changed](https://github.com/tobymao/sqlglot/commit/1c67f030cd9df530e26c620079b2298b1db97d50) code wasn't reached.